lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
The point is that you map a field to use localized property name encoding. If you save without a locale, here, then any multi language property you have set will not be persisted.

Swallowing exceptions is not the way to handle this, the logic, whatever it is, should b explicit, you can see that this is bad as the exception swallowing behavior is duplicated multiple times.

For the sake of simplicty and clarity and predictability I say that the locale should be mandatory, defauting to the default locale. This saves lots of trouble and will make the system more stable - this is the way the PHPCR-ODM works, and I would bet it is how the Doctrine Extensions work also.

If you load a document, it will be loaded in the default locale. If you save it (without a locale) it would be saved in that same locale.

What issues arrise by using the default locale?

I appreciate that none of you *like* this behavior, but that doesn't make it invalid.